### DISCUSSION TITLE
Upgrading coc-nvim

### DISCUSSION THREAD

**Discussion by omercsp** [neoclide/coc.nvim#2806] [Q&A]
I couldn't find any documentation about the upgrading the plugin itself.

1. Is there any indication that an updated version of coc-nvim is available?
2. Assuming that I know such a version exist, - how do I install it?

Thanks

**Comment by chemzqm** [comment]
Read doc of plugin manager you'reusing

**Comment by omercsp** [comment]
Well, I'm using VIM8's native plugin. Normally it is just a matter of updating the files in the plugin folder, but since coc-nvim is a big one, that requires nodejs packages and such, I figured there might be additional steps to do a proper upgrade. 

If not - great. It's worth mentioning in the wiki that upgrading requires only redeploying the plugin files.

**Comment by chemzqm** [comment]
Release branch contains code that already compiled, no extra steps needed for update.
